src/server.js

```diff
@@ -13,43 +13,65 @@ const initServer = (config) => {
 
   server.route([
     {
-      method: 'POST',
-      path: '/github',
+      method: "POST",
+      path: "/github",
       options: {
         validate: {
-          payload: joi.object({
-            hook: joi.object({ "events": joi.array().items(joi.string()) }).required().unknown(),
-            sender: joi.object({login: joi.string().required()}).required().unknown(),
-            repository: joi.object({full_name: joi.string().required()}).required().unknown(),
-          }).unknown(),
+          headers: joi.object({ "x-github-event": joi.string().required() }).unknown(),
+          payload: joi
+            .object({
+              action: joi.string(),
+              hook: joi
+                .object({ events: joi.array().items(joi.string()) })
+                .unknown(),
+              sender: joi
+                .object({ login: joi.string().required() })
+                .required()
+                .unknown(),
+              repository: joi
+                .object({ full_name: joi.string().required() })
+                .required()
+                .unknown(),
+            })
+            .unknown(),
         },
       },
       handler: async (request, h) => {
+        const { payload, headers } = request;
+        const event = headers["x-github-event"];
         const {
-          hook: { events },
-          sender: {
-            login: senderLogin
-          },
-          repository: {
-            full_name: repositoryFullName
-          },
-        } = request.payload;
-
-        if (events.includes('star')) {
+          repository: { full_name: repositoryFullName },
+        } = payload;
+
+        if (event === "ping" && request.payload.hook.events.includes("star")) {
+          await axios.post(STREAMLABS_ENDPOINT, {
+            access_token: config.STREAMLABS_TOKEN,
+            type: "follow",
+            message: `Configured *${repositoryFullName}*`,

```

Comment:
Maybe we could specify a bit more in this message:
- The repository has been correctly configured to send webhook requests
- The webhook includes `star` events
- The `STREAMLABS_TOKEN` is correct because you're seeing this message